cmd/minikube/cmd/docker-env.go

```diff
@@ -119,6 +122,12 @@ func isDockerActive(r command.Runner) bool {
 	return sysinit.New(r).Active("docker")
 }
 
+func restartOrExitDaemon(bin string, name string, runner command.Runner) {
```

Comment:
nit: For idiomatic go, functions which may cause an exit should be named something like `mustRestartDocker`.

Reply:
done